.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'issue title'
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
+
+Output of helm version:
+
+Output of kubectl version:
+
+Cloud Provider/Platform (AKS, GKE, Minikube etc.):
+
+
+
+<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. 
+
+It's helpful to search the existing GitHub issues first.
+It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
+
+Fill in as much of the template below as you can.  If you leave out information, we can't help you as well.
+
+Be ready for followup questions, and please respond in a timely manner. If we can't reproduce a bug or think a feature already exists, we might close your issue.  If we're wrong, PLEASE feel free to reopen it and explain why.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Version of Helm, Kubernetes, kubectl**:
```

Comment:
It looks like we ask for the version of Helm and Kubernetes and kubectl twice. Once above and here again.

Reply:
thanks, I've removed